### PR TITLE
Add content/pt-br/docs/concepts/policy/node-resource-managers.md

### PR DESCRIPTION
**Pull request by castroricardo1**
my pr was closed because the repository was deleted, this pr already has the updates suggested by the reviews

https://github.com/kubernetes/website/pull/38080

### CODE REVIEW

File: content/pt-br/docs/concepts/policy/node-resource-managers.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Gerenciadores de Recursos de Nós
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+Para dar suporte a cargas de trabalho com latência crítica e altas taxa de transferência, o Kubernetes oferece um conjunto de gerenciadores de recursos. Os gerenciadores visam coordenar e otimizar o alinhamento de recursos do nó(s) para Pods configurados com um requisito específico para CPUs, dispositivos e recursos de memória (hugepages).
```
Comment (stormqueen1990) [inline content/pt-br/docs/concepts/policy/node-resource-managers.md:9]: Minor: "taxa" should be plural.

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @castroricardo1!

I just found a minor typo (a word that is singular but should be plural). Other than that, it's a LGTM.

/cc @MrErlison
Could you please provide a second review?

File: content/pt-br/docs/concepts/policy/node-resource-managers.md (hunk 1 above)
Comment (castroricardo1) [reply content/pt-br/docs/concepts/policy/node-resource-managers.md]: Ok, fixed

**Comment by MrErlison** [comment]
Hi @castroricardo1 Thank you for your contribution!

/lgtm

**Comment by stormqueen1990** [comment]
/approve

Thank you very much! 😃

**Pull request merged**